pkg/types/types.go

```diff
@@ -19,6 +21,9 @@ const (
 
 	SupportBundleManager = "support-bundle-manager"
 	SupportBundleAgent   = "support-bundle-agent"
+
+	PodCreationTimeout      = time.Minute
```

Comment:
Not quite sure 1 m is good for timeout, but we can see how it will work out.

Reply:
Not expecting the DaemonSet Pod scheduling to be quick. But since we don't know, so I've increased it to 5 mins just in case.

Reply:
Ideally, it should be very quick, since it's about scheduling instead of waiting for pods to run.

The other you can do is if the number of pods does not match the desired number of DS.
- dump an obvious warning log 
- still return the scheduled node list, so it's the best effort instead of failing the support bundle generation.

Reply:
The current implementation of the node collection is best effort.

Ref: https://github.com/rancher/support-bundle-kit/blob/e459c3b61067a5938350f7acff6ed5c02192fc05/pkg/manager/manager.go#L208-L213

Reply:
Good